connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -579,6 +579,27 @@ message ArtifactStatusesResponse {
   map<string, ArtifactStatus> statuses = 1;
 }
 
+message InterruptRequest {
```

Comment:
should `InterruptRequest` have an enum for interrupt type? All etc? We can start with `UNSET/ ALL` and expand from there?

Reply:
Done